react/components/PriceRange/index.js

```diff
@@ -20,13 +23,28 @@ const PriceRange = ({ title, facets, priceRange, priceRangeLayout }) => {
   const intl = useIntl()
   const navigateTimeoutId = useRef()
 
+  const { push } = usePixel()
+  const { searchQuery } = useSearchPage()
+
+  const pushPixelEvent = (left, right) => {
```

Comment:
what does this left rigth means?

Reply:
Hi Igor, left and right means the bound left and bound right of range choose by user